terraform/modules/webservers/webservers.tf

```diff
@@ -1,6 +1,7 @@
 # Required variables
 variable "environment" { type = "string" }
 variable "vpc_id" { type = "string" }
+variable "subnet" { type = "string" }
```

Comment:
This isn't necessary. subnets is calculated for you at line 35. See example usage at line 132.